### DISCUSSION TITLE
Axum Header Delimiter

### DISCUSSION THREAD

**Discussion by joelawm** [tokio-rs/axum#1841] [Q&A]
Hello Axum Community so I have been working on converting my old Rocket backend to Axum and have been having a blast. I noticed some weird anomaly while adding a sub request feature with Nginx into my microservice application.
I have this little piece of code that I parse into a i32 to pass along the id of the user but before I do that I have to remove the delimiter from the original request.
`headers.get("X-Authentication-Id").map(|x| x.to_str())`

So I use this to remove the ': ' from the header
`// Remove the delimiter ': ' form the string
let removed = context.to_string().replace(": ", "");`

I haven't had to do this for any of the other headers, is this normal?

**Comment by davidpdrsn** [comment]
I think you to share what your code looks like this, because I don't quite follow what you mean. What do you have to remove `:` from? The header key or value? What is `context`?
